runtime/vendored/asserts.bundle.js

```diff
@@ -0,0 +1,808 @@
+// deno-fmt-ignore-file
+// deno-lint-ignore-file
+// This code was bundled using `deno bundle` and it's not recommended to edit it manually
+
+const { Deno  } = globalThis;
+const noColor = typeof Deno?.noColor === "boolean" ? Deno.noColor : true;
```

Comment:
I would love to enable coloured errors and coloured output from other Deno libs in general.

However, we decided to hide all Deno APIs.

What do you think @juliangruber @patrickwoodhead about exposing a very limited subset of Deno APIs, starting with `Deno.noColor`?

If we claim that Zinnia is mostly compatible with Deno, then I think this will be reasonable and justifiable.

Reply:
To me colored output isn't important enough to complicate the implementation, at this stage of the project. But if it's easy to add, why not

Reply:
I created #49. You are right that this is not a priority now.